EU4toV2/Source/EU4World/EU4Country.h

```diff
@@ -174,6 +176,8 @@ namespace EU4
 			bool							possibleShogun;			// if this country is the shogun
 			std::vector<std::shared_ptr<EU4::leader>> militaryLeaders;
 			string							government;				// the government type
+			int								governmentRank;		// Government rank, relevant for empire cultural unions
+			int								development;			// Development realm size, relevant for empire cultural unions
```

Comment:
The surrounding code is a bad example, but for new code:
1. have just a space between the type and name.
2. Have a default value for non-constructed types (like int).
3. No need for the comment, the name should be sufficient to explain.

Reply:
removed spaces, dropped comments, included default value.